exporter/otlp-http/lib/opentelemetry/exporter/otlp/http/trace_exporter.rb

```diff
@@ -230,14 +230,14 @@ def measure_request_duration
             ensure
               stop = Process.clock_gettime(Process::CLOCK_MONOTONIC)
               duration_ms = 1000.0 * (stop - start)
-              @metrics_reporter.record_value('otel.otlp_exporter.request_duration',
+              @metrics_reporter.record_value(SemanticConventions::Common::OTEL_OTLP_EXPORTER_REQUEST_DURATION,
```

Comment:
This constant is defined in `...::Attributes` rather than `SemanticConventions::Common::`.